packages/pigz/pigz-2.6_Makefile.patch

```diff
@@ -0,0 +1,20 @@
+diff -u pigz-2.6.orig/Makefile pigz-2.6/Makefile
```

Comment:
`pigz` already has a patch.

https://github.com/vhqtvn/termux-packages/blob/for-merge-20210528/packages/pigz/Makefile.patch